matrix/common/aws/cloudwatch_handler.py

```diff
@@ -0,0 +1,40 @@
+import os
+from enum import Enum
+
+import boto3
+
+
+class MetricName(Enum):
+    """
+    MetricNames for cloudwatch custom metric datapoints.
+    """
+    REQUEST = "Matrix Request"
+    REQUEST_COMPLETION = "Matrix Request Completion"
+    REQUEST_ERROR = "Matrix Request Error"
```

Comment:
Though we do plan on replacing `RequestTracker.log_error` in the future, that could be a good place to start counting error metrics